addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -173,6 +183,33 @@ Wysiwyg.include({
             return this._super.apply(this, arguments);
         }
     },
+
+    /**
+     * When the editable is inside an iframe, use the iframe element instead
+     * of the editable to check whether it is part of the subtree.
+     *
+     * @override
+     **/
+    _isEditableInSubTree: function (root) {
+        if (this.options.inIframe) {
+            return root.contains(this.$iframe[0]);
+        } else {
+            return this._super.apply(this, arguments);
+        }
+    },
+
+    /**
+     * @override
+     **/
+    _configureToolbar: function () {
+        this._super.apply(this, arguments);
+        if (this.options.inIframe && !this.options.snippets) {
+            this._onScrollIframe = () => {
```

Comment:
I think that with the construction I suggest above, you won't need a new function here. You could reuse `this._onScroll`. You'll still need to attach it to the other document here, though.

Reply:
Ok. A tiny bit of logic had to be included in `onScroll` in order to determine which document fired the event.

Reply:
I accidentaly rebased before pushing the changes... That means the diff of the force-push will be useless to you. My apologies for that...